mage/k3d/lib.go

```diff
@@ -66,9 +68,43 @@ func createCluster() {
 	mageutil.PanicOnError(err)
 }
 
+func withExponentialBackoff(f func() error, baseWait, timeout time.Duration) error {
```

Comment:
I recommend using https://github.com/cenkalti/backoff instead of rolling our own

Reply:
I took this out as we are updating k3d